jobs/wf1-2-createPatients.js

```diff
@@ -116,3 +116,6 @@ each('patients[*]', state => {
     return state;
   });
 });
+
+// Clean up state
+fn(state => ({ ...state, data: {}, references: [] }));
```

Comment:
I've just noticed that there are a couple of operations in this file that need the `(state)` wrapper removing (the each/post on line 35, the each/createPatient on line 106)